evm_loader/lib/src/types/tracer_ch_common.rs

```diff
@@ -129,3 +138,65 @@ pub struct EthSyncing {
     pub current_block: u64,
     pub highest_block: u64,
 }
+
+pub struct RevisionMap {
+    map: BTreeMap<u64, String>,
+    pub last_update: Instant,
```

Comment:
Nitpick: perhaps it is better to remove `pub` here and add a getter method. Personally I don't like mixing public and private methods in the same structure, but it is up to you.

Reply:
In my opinion, this is not considered bad practice if we don't have a lot of fields in the structure.